computer/revere-rhino-x86_64-1/setup.sh

```diff
@@ -29,8 +29,10 @@ sed -i "$sed_arg" install-conf.sh
 
 sed_arg="s/subnet=.*/subnet=10.40.40.0/; \
 	s/dhcp_clients=.*/dhcp_clients=( \
-	  'revere-rhino-x86_64-2,00:24:9b:15:4a:ea,40', \
+	  'revere-rhino-x86_64-2,50:3e:aa:b7:7c:a0,40', \
 	  'axis-m1124-0,ac:cc:8e:23:6e:8d,50', \
+	  'dalsa-c2450-0,00:01:0d:c3:84:f8,60', \
```

Comment:
I don't know the MAC addresses

Reply:
I have verified & tested this personally.